mirror-godot-app/scripts/autoload/bootup.gd

```diff
@@ -34,9 +34,6 @@ func _start_client():
 	# make request for version from server
 	Net.version_client.get_client_version()
 	var version = await Net.version_client.version_received
-	if version != Util.get_version_string():
```

Comment:
this should be done using a setting not removing the code since there is another app depending upon this functionality.

Reply:
Fixed